doc/api/console.md

```diff
@@ -73,6 +73,8 @@ const Console = console.Console;
 ```
 
 ### new Console(stdout[, stderr])
+* ```stdout``` {Writable Stream}
```

Comment:
The actual class should be `Writable`, which is also used by documentations of `repl` and `readline`.

(Also, why triple backticks? Not that it really matters though, just out of curiosity.)